scripts/pullrequest/update.sh

```diff
@@ -13,4 +13,5 @@ set +o allexport
 docker compose \
   -f docker-compose.yml \
   -f docker-compose.pizza.yml \
+  -f docker-compose.seed.yml \
```

Comment:
Will this overwrite all Pizza data when a new commit is pushed? 

I think the intention would be that the Pizza is just seeded once on creation and then data will persist until the Vultr instance is destroyed / recreated manually if we need to re-seed.

Reply:
The `ON CONFLICT DO NOTHING` bit is what I forgot! This looks good, thanks for explaining.